nginx/default

```diff
@@ -6,8 +6,8 @@ server {
     # Make site accessible from http://set-ip-address.xip.io
     server_name localhost;
 
-    access_log /var/log/nginx/satisfy-access.log;
-    error_log  /var/log/nginx/satisfy-error.log error;
+    access_log /proc/1/fd/1;
+    error_log  /proc/1/fd/2 error;
```

Comment:
Are you sure that error logs AND access logs are being redirected to stderr and stdout. Because once I had some problems redirecting access_log to stdout and I had to redirect it to stderr too (`/proc/1/fd/2`).

Reply:
PID 1 is the process launched by docker, and its STDOUT will be what docker picks up. NGINX has a new Process ID. /dev/stdout and /dev/stderr didn't work in that case.